workflow/tests/hpxml_translator_test.rb

```diff
@@ -1358,15 +1358,17 @@ def skip_utility_bill_warning(err_line)
       assert_operator(unmet_hours_htg, :>, 1000)
       assert_operator(unmet_hours_clg, :>, 1000)
     else
-      if hpxml.total_fraction_heat_load_served == 0
-        assert_equal(0, unmet_hours_htg)
-      else
-        assert_operator(unmet_hours_htg, :<, 350)
-      end
-      if hpxml.total_fraction_cool_load_served == 0
-        assert_equal(0, unmet_hours_clg)
-      else
-        assert_operator(unmet_hours_clg, :<, 350)
+      if not hpxml_path.include? 'base-hvac-onoff-thermostat-deadband.xml'
```

Comment:
Can we update the unmet hours EMS program to properly account for the deadband temperature?

Reply:
My concern is that the E+ approach of modifying setpoint during the on/off thermostat operations may make unmet loads challenging to make sense, eg. when hvac is off, the thermostat setpoint is set to be the lower end of deadband and if the hvac is on, the thermostat setpoint is set to be the higher end of deadband. It could make every cycle an unmet loads. This is from what I realized that what happened in E+, need to double check the `Zone Heating/Cooling Setpoint Not Met Time` outputs to confirm.

Reply:
Right. The idea is that we can zero out unmet loads in our [EMS program](https://github.com/NREL/OpenStudio-HPXML/blob/4c519231ee9604d0baaa8ee0a840b190559b8ee8/HPXMLtoOpenStudio/measure.rb#L1917-L1941) for times when the indoor temperature is actually meeting the nominal (i.e., not adjusted for deadband) setpoint, so that it won't be confusing to the user.

Reply:
(In the same way that we currently zero out unmet loads when they occur outside the heating/cooling season.)